addons/base_geolocalize/models/res_partner.py

```diff
@@ -32,9 +32,12 @@ def _geo_localize(self, street='', zip='', city='', state='', country=''):
 
     def geo_localize(self):
         # We need country names in English below
-        if not self._context.get('force_geo_localize') \
-                and (self._context.get('import_file') \
-                     or any(config[key] for key in ['test_enable', 'test_file', 'init', 'update'])):
+        if not self._context.get('force_geo_localize') and (
+            self._context.get('import_file')
+            or modules.module.current_test
+            or config['test_enable']
+            or any(config[key] for key in ['init', 'update'])
+        ):
```

Comment:
```suggestion
        if not self._context.get('force_geo_localize') and (
            self._context.get('import_file')
            or modules.module.current_test
            or not self.env.registry.ready
        ):
```

maybe something like that  (not sure, to check and test)